interface/forms/prior_auth/FormPriorAuth.class.php

```diff
@@ -125,4 +132,24 @@ function get_date()
     {
         return $this->date;
     }
+
+    function get_date_from()
+    {
+        return oeFormatShortDate($this->date_from);
+    }
+
+    function set_date_from($dt)
+    {
+        $this->date_from = DateToYYYYMMDD($dt);
+    }
+
+    function get_date_to()
+    {
+        return oeFormatShortDate($this->date_to);
+    }
+
+    function set_date_to($dt)
+    {
+        $this->date_to = DateToYYYYMMDD($dt);
+    }
```

Comment:
not sure if this will work as expected for internationalization (since i think these functions are called when getting the data from database to hydrate the class). rec. testing it with different date formats in globals to ensure no odd behavior.

Reply:
good pt, that was a long shot, when tested with MM/DD/YYYY it failed and then saved as 0000-00-00 :(
back to the drawing board

Reply:
strategy would be to get the DateToYYYYMMDD() when it saves (look into save.php script).
can just pipe the oeFormatShortDate call when output it in smarty template (look something like `<input type='text' size='10' class='form-control datetimepicker' name='date_from' id='date_from' value='{$prior_auth->get_date_from()|oeFormatShortDate|attr}' />`)

Reply:
at line 21 in save.php would do something like:
```
$_POST['from_date']  = (isset($_POST['from_date'])) ? DateToYYYYMMDD($_POST['from_date']) : '';
```

Reply:
actually may be better to use null:
```
$_POST['from_date']  = (!empty($_POST['from_date'])) ? DateToYYYYMMDD($_POST['from_date']) : null;
```